lib/dynamic_widget/basic/textfield_widget_parser.dart

```diff
@@ -2,7 +2,14 @@ import 'package:dynamic_widget/dynamic_widget.dart';
 import 'package:dynamic_widget/dynamic_widget/utils.dart';
 import 'package:flutter/material.dart';
 
+import '../../new_widget_parser.dart';
+
 class TextFieldWidgetParser extends NewWidgetParser {
+  @override
+  void assertionChecks(Map<String, dynamic> map) {
+    // TODO: implement assertionChecks
```

Comment:
can we add assertions here, we will be needing it in this page

Reply:
yeah thats why added this method to be override because we want assertions in every widget parser. Adding to TextField now